easybuild/easyconfigs/j/jupyter-server-proxy/jupyter-server-proxy-3.2.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,73 @@
+easyblock = 'PythonBundle'
+
+name = 'jupyter-server-proxy'
+version = '3.2.1'
+
+homepage = 'https://jupyter.org'
+description = """JupyterHub is a multiuser version of the Jupyter (IPython) notebook designed
+ for centralized deployments in companies, university classrooms and research labs."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+]
+dependencies = [
+    ('Python', '3.9.5'),
+    ('JupyterHub', '1.4.1'),
+    ('OpenSSL', '1.1', '', True),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('sniffio', '1.2.0', {
+        'checksums': ['c4666eecec1d3f50960c6bdf61ab7bc350648da6c126e3cf6898d8cd4ddcd3de'],
+    }),
+    ('anyio', version, {
+        'checksums': ['07968db9fa7c1ca5435a133dc62f988d84ef78e1d9b22814a59d1c62618afbc5'],
+    }),
+    ('requests-unixsocket', '0.2.0', {
+        'checksums': ['9e5c1a20afc3cf786197ae59c79bcdb0e7565f218f27df5f891307ee8817c1ea'],
+    }),
+    ('websocket-client', '1.1.0', {
+        'modulename': 'websocket',
+        'checksums': ['b68e4959d704768fa20e35c9d508c8dc2bbc041fd8d267c0d7345cffe2824568'],
+    }),
+    ('jupyter-server', '1.9.0', {
+        'source_tmpl': 'jupyter_server-%(version)s.tar.gz',
+        'checksums': ['7d19006380f6217458a9db309b54e3dab87ced6c06329c61823907bef2a6f51b'],
+    }),
+    ('simpervisor', '0.4', {
+        'checksums': ['cec79e13cdbd6edb04a5c98c1ff8d4bd9713e706c069226909a1ef0e89d393c5'],
+    }),
+    ('multidict', '5.1.0', {
+        'checksums': ['25b4e5f22d3a37ddf3effc0710ba692cfc792c2b9edfb9c05aefe823256e84d5'],
+    }),
+    ('async-timeout', '3.0.1', {
+        'checksums': ['0c3c816a028d47f659d6ff5c745cb2acf1f966da1fe5c19c77a70282b25f4c5f'],
+    }),
+    ('typing-extensions', '3.10.0.0', {
+        'source_tmpl': 'typing_extensions-%(version)s.tar.gz',
+        'checksums': ['50b6f157849174217d0656f99dc82fe932884fb250826c18350e159ec6cdf342'],
+    }),
+    ('yarl', '1.6.3', {
+        'checksums': ['8a9066529240171b68893d60dca86a763eae2139dd42f42106b03cf4b426bf10'],
+    }),
+    ('aiohttp', '3.7.4.post0', {
```

Comment:
We should make this depend on `aiohttp` from #14784 . It would simplify this easyconfig.

Reply:
Done